src/stratigraphy/util/draw.py

```diff
@@ -61,6 +63,16 @@ def draw_predictions(predictions: list[FilePredictions], directory: Path, out_di
                         logger.warning("MLFlow could not be imported. Skipping logging of artifact.")
 
 
+def draw_coordinates(page: fitz.Page, coordinates: list[float]) -> None:
+    """Draw the coordinates on a pdf page.
+
+    Args:
+        page (fitz.Page): The page to draw on.
+        coordinates (list[float]): List of coordinates to draw.
+    """
+    page.add_freetext_annot(fitz.Rect([5, 5, 100, 250]), f"Coordinates: {coordinates}")
```

Comment:
I'm not a huge fan of this way of including the coordinates on the PNG image. If they are printed on a dark background, then you can hardly read them. And this does not give any information about where on the page the coordinates where found, which is kind-of the whole point of creating this PNG visualisation in the first place. It's not a major issue, we can also leave it like this for now, but I think we should think about how to better visualise the coordinates (and other extracted metadata) in the future.

Reply:
As discussed. Let's keep it for now.